pkg/ddl/backfilling.go

```diff
@@ -168,26 +167,40 @@ type backfillCtx struct {
 }
 
 func newBackfillCtx(id int, rInfo *reorgInfo,
-	schemaName string, tbl table.Table, jobCtx *ReorgContext, label string, isDistributed bool) (*backfillCtx, error) {
-	// TODO: remove newReorgSessCtx
-	sessCtx := newReorgSessCtx(rInfo.jobCtx.store)
-	if err := initSessCtx(sessCtx, rInfo.ReorgMeta); err != nil {
+	schemaName string, tbl table.Table, jobCtx *ReorgContext, label string, isDistributed bool, isUpdateColumn bool) (*backfillCtx, error) {
+	warnHandler := contextutil.NewStaticWarnHandler(0)
+	exprCtx, err := newReorgExprCtxWithReorgMeta(rInfo.ReorgMeta, warnHandler)
+	if err != nil {
 		return nil, errors.Trace(err)
 	}
 
+	if isUpdateColumn {
```

Comment:
These logic is from: https://github.com/pingcap/tidb/pull/55873/files#diff-f33325470a81ba90410cdde792a53e6f6a1af0ec43baa840164a37dd9d2739a0L614-L619